code/renderers/react/src/portable-stories.tsx

```diff
@@ -54,9 +55,66 @@ export function setProjectAnnotations(
 // This will not be necessary once we have auto preset loading
 export const INTERNAL_DEFAULT_PROJECT_ANNOTATIONS: ProjectAnnotations<ReactRenderer> = {
   ...reactProjectAnnotations,
-  renderToCanvas: (renderContext, canvasElement) => {
+  beforeAll: async function reactBeforeAll() {
+    try {
+      // copied from
+      // https://github.com/testing-library/react-testing-library/blob/3dcd8a9649e25054c0e650d95fca2317b7008576/src/pure.js
+      const { configure } = await import('@storybook/test');
+
+      configure({
+        unstable_advanceTimersWrapper: (cb) => {
+          return act(cb);
+        },
+        asyncWrapper: async (cb) => {
+          const previousActEnvironment = getReactActEnvironment();
+          setReactActEnvironment(false);
+          try {
+            const result = await cb();
+            // Drain microtask queue.
+            // Otherwise we'll restore the previous act() environment, before we resolve the `waitFor` call.
+            // The caller would have no chance to wrap the in-flight Promises in `act()`
+            await new Promise<void>((resolve) => {
+              setTimeout(() => {
+                resolve();
+              }, 0);
+
+              if (jestFakeTimersAreEnabled()) {
+                // @ts-expect-error global jest
+                jest.advanceTimersByTime(0);
+              }
+            });
+
+            return result;
+          } finally {
+            setReactActEnvironment(previousActEnvironment);
+          }
+        },
+        eventWrapper: (cb) => {
+          let result;
+          act(() => {
+            result = cb();
+          });
+          return result;
+        },
+      });
+    } catch (e) {
+      console.log(e);
```

Comment:
if the dependency is optional, does it make sense to log any error? Shouldn't you log the error only if the dependency exists?